website/docs/r/cognitive_account.html.markdown

```diff
@@ -86,7 +86,7 @@ The following arguments are supported:
 
 * `outbound_network_access_restricted` - (Optional) Whether outbound network access is restricted for the Cognitive Account. Defaults to `false`.
 
-* `project_management_enabled` - (Optional) Whether project management is enabled when the `kind` is set to `AIServices`. Once enabled, `project_management_enabled` cannot be disabled. Changing this forces a new resource to be created. Defaults to `false`.
+* `project_management_enabled` - (Optional) Whether project management is enabled when the `kind` is set to `AIServices`. Changing this forces a new resource to be created except when rolling back from `kind` `AIServices` to `OpenAI`. Defaults to `false`.
```

Comment:
This should be updated because we allow update `project_management_enabled` to `true` within the `AIServices` kind.